frontend/src/views/instructors/delete-instructor-dialog.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { connect } from "react-redux";
+import { deleteInstructor, instructorsSelector } from "../../api/actions";
+import { Modal, Button } from "react-bootstrap";
+
+function DeleteInstructorDialog(props) {
+    const { show, onHide = () => {}, id, deleteInstructor } = props;
+    function removeInstructor() {
+        deleteInstructor({ id });
```

Comment:
is this the deleteInstructor from props or the one that is imported on line 3?

Reply:
From props